css/css-typed-om/historical.html

```diff
@@ -10,4 +10,13 @@
   // https://github.com/w3c/css-houdini-drafts/commit/5261c1a323ea062d69cb5a3f1e69734fd176948a
   assert_false("CSSPositionValue" in self);
 }, "Support for CSSPositionValue");
+
+test(function() {
+    // https://github.com/w3c/css-houdini-drafts/commit/5a1663db3f9c36ffa7b2013afdd345e98ae03d5e
+    // https://github.com/w3c/css-houdini-drafts/commit/223258ea4c71aa60b1bd675602dddfbb414e0481
+    const deprecatedProperties = ["toRGB", "toHSL", "toHWB", "toGray", "toLCH", "toLab", "toColor", "toDeviceCMYK", "to", "colorSpace"];
+    for (const deprecatedProperty of deprecatedProperties) {
```

Comment:
Let's move this loop outside of the `test`, so we produce one subtest for each of them. Probably overkill, but it doesn't hurt to make it obvious what's failing. (I wouldn't really hold you to this, if it weren't for the bigger change needed!)